src/settings/SettingsStore.js

```diff
@@ -370,6 +370,20 @@ export default class SettingsStore {
         return SettingsStore._getFinalValue(setting, level, roomId, null, null);
     }
 
+    /**
+     * Gets the default value of a setting.
+     * @param {string} settingName The name of the setting to read the value of.
+     * @return {*} The value, or null if not found
```

Comment:
It throws if not found.
```suggestion
     * @return {*} The default value
```